src/Illuminate/Auth/AuthenticationException.php

```diff
@@ -3,16 +3,26 @@
 namespace Illuminate\Auth;
 
 use Exception;
+use Illuminate\Support\Collection;
 
 class AuthenticationException extends Exception
 {
+    protected $guards = [];
+
     /**
      * Create a new authentication exception.
      *
-     * @param string  $message
+     * @param string|array $guards
+     * @param string $message
      */
-    public function __construct($message = 'Unauthenticated.')
+    public function __construct($guards = [], $message = 'Unauthenticated.')
```

Comment:
Good point - didnt think that people could be throwing the AuthenticationException 'manually'.

Perhaps moving the $guards param to the 2nd slot on the constructor would be ok?